docs/architecture/MLOps_process.md

```diff
@@ -68,7 +68,7 @@ Initial experiments can be done in a notebook, however, there are several signs
 
 Once you see any of these signs, it’s time to wrap experiments using Azure ML Pipeline SDK. We would recommend using the three-step approach:
 
-* **Step 1. Refactor the notebook to Python files.** The primary goal of this step is to move all methods/classes to separate Python files and make them independent from the execution environment.
+* **Step 1. Refactor the notebook to Python files.** This step focuses on pulling out and generalizing components which you expect to be reused across multiple experiments, allowing future notebooks to be implemented more easily using the refactored Python code.
```

Comment:
Definitely have @sbaidachni review this change. I meant _in addition to_ the text you already had there, so I'm not sure if this alters the original intent.

Reply:
Looks great